qiskit/pulse/builder.py

```diff
@@ -1636,9 +1636,11 @@ def barrier(*channels_or_qubits: Union[chans.Channel, int]):
 
 
 # Macros
-def measure(qubit: int,
-            register: Union[chans.MemorySlot, chans.RegisterSlot] = None,
-            ) -> Union[chans.MemorySlot, chans.RegisterSlot]:
+def measure(qubits: Union[List[int], int],
+            registers: Union[List[Union[chans.MemorySlot, chans.RegisterSlot]],
```

Comment:
These types are getting excessively long. You could define a new type above. https://docs.python.org/3/library/typing.html#type-aliases

Reply:
done. any suggestion for type name? I defined `MeasResource` to express register + memory slot.

Reply:
`MeasLoc`, `MeasurementLocation`, `StorageLocation` ?

Reply:
`StorageLocation` is much better. thanks